src/unix/stream.c

```diff
@@ -301,7 +301,9 @@ int uv__stream_try_select(uv_stream_t* stream, int* fd) {
   if (fds[1] > max_fd)
     max_fd = fds[1];
 
-  sread_sz = (max_fd + NBBY) / NBBY;
+  sread_sz = (max_fd + 1) / NBBY;
+  if (sread_sz % sizeof(uint32_t) != 0)
+    sread_sz += sizeof(uint32_t) - (sread_sz % sizeof(uint32_t));
```

Comment:
@saghul Done.
@bnoordhuis Sorry, we didn't get rid of the `NBBY` usage after all; we still need to convert the number of bits into bytes, so we can then round that up to a muliple of `sizeof(guint32)`. :disappointed: